src/coldfront_plugin_api/urls.py

```diff
@@ -8,7 +8,6 @@
 from coldfront_plugin_api import auth, serializers
 from coldfront_plugin_api.scim_v2 import groups, users
 
-
```

Comment:
You have remove whitespace here by mistake, your Pull Requests shouldn't introduce unrelated changes like this.